lib/winston/logger.js

```diff
@@ -231,7 +217,8 @@ class Logger extends Transform {
       const tokens = msg && msg.match && msg.match(formatRegExp);
 
       if (!tokens) {
-        const info = Object.assign({}, this.defaultMeta, meta, {
+        const infoObj = { message: msg };
```

Comment:
Why is this necessary?  (In particular, why do we frontload `message`, but not e.g. `[SPLAT]`, `level`, etc.?)

Reply:
Can you elaborate on this? Not sure I understand what you mean in regards to frontloading the `message` in relation to the other information.